include/LLVMSPIRVLib.h

```diff
@@ -137,6 +137,17 @@ using SpecConstInfoTy = std::pair<uint32_t, uint32_t>;
 bool getSpecConstInfo(std::istream &IS,
                       std::vector<SpecConstInfoTy> &SpecConstInfo);
 
+struct SPIRVHeaderData {
+  std::uint32_t MemoryModel = 0x7fffffff; // MemoryModelMax
+  std::uint32_t Version = 0;
```

Comment:
We should have `SPIRV::VersionNumber` available here I think?  So we could use that instead of `uint32_t`.